src/js/models/tool.js

```diff
@@ -42,7 +42,7 @@ define(function (require, exports, module) {
         this.id = id;
         this.icon = id;
         this.name = name;
-        this.nativeToolName = nativeToolName;
+        this.nativeToolName = function () { return nativeToolName; };
```

Comment:
the super select native tool name is now a function that returns either direct select or the move tool depending on context.